Arcade/ASTEROIDS/game.ini

```diff
@@ -0,0 +1,5 @@
+title=ASTEROIDS 
+date=2018-06-28T04:41:00Z 
+author=CDR_Xavier 
+description=The unique 2D space shooter. Ammo in the bottomless box.
+license=GNL-2.0
```

Comment:
Still not right.
Now it says `GNL-2.0`, but it needs to say `GPL-2.0`.
It is the GNU **G**eneral **P**ublic **L**icence version **2.0**, hence `GPL-2.0`.